data/tests.yml

```diff
@@ -794,6 +795,7 @@
 - description: bus:lanes:backward=designated
   skip_python: true
   skip_kotlin: true
+  rust: false # fails roundtrip (outputs busway:right=opposite_lane tag)
```

Comment:
Can you paste the full error here?
I don't particularly like taking a step backward in passing tests :P

Reply:
I just get this now, which is different to before. I should be able to look into it later, as `set_busway_scheme` is supposed to work with the checks parsing `LanesScheme`. Perhaps more of those checks between schemes that I had as TODO will illuminate the problem... 

```
running 6 tests
test metric::speed::tests::test_speed ... ok
test locale::tests::test_locale ... ok
test metric::tests::test_speed ... ok
test tag::tests::test_tags ... ok
test test::tests::test_roundtrip ... FAILED
test test::tests::test_from_data ... ok

failures:

---- test::tests::test_roundtrip stdout ----
thread 'test::tests::test_roundtrip' panicked at 'called `Result::unwrap()` on an `Err` value: Msg(Unsupported { description: Some("no backward lanes for busway"), tags: None })', osm2lanes/src/test.rs:480:18
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
oh yeah, this is the the whole "we need a backward lane to turn into a busway problem".

can we modify the business logic in the edge case to switch from the outside backward lane to the inside forward lane in this case?